exporter/kineticaexporter/config.go

```diff
@@ -34,7 +53,196 @@ func (cfg *Config) Validate() error {
 		return errors.New("Protocol must be either `http` or `https`")
 	}
 
+	if !fileExists(cfg.LogConfigFile) {
+		fmt.Println("WARNING : LOG config file ", cfg.LogConfigFile, " does not exist; will work with default logger ...")
+	}
+
 	return nil
 }
 
+func parseNumber(s string, fallback int) int {
+	v, err := strconv.Atoi(s)
+	if err == nil {
+		return v
+	}
+	return fallback
+}
+
+// createLogger - creates a Logger using user defined configuration from a file.
+// If the file name is not found or there is some error it will create a default
+// logger using the default logger config file named 'config_log_zap.yaml'.
+//
+//	@receiver cfg
+//	@return *zap.Logger
+func (cfg *Config) createLogger() *zap.Logger {
```

Comment:
Unless it is required for exporting, we should not create a specific `zap.Logger`.  Instead, we should use the logger supplied in `set exporter.CreateSettings`.

Reply:
What if someone wants to customize file based logging with rollover, logging formats etc. ?

Reply:
At the moment the collector doesn't support that. It does expose these options to users for its logger: https://github.com/open-telemetry/opentelemetry-collector/blob/7c5ecef11dff4ce5501c9683b277a25a61ea0f1a/service/telemetry/config.go#L33

Reply:
I am working on removing this.